### DISCUSSION TITLE
Выбор прокси для приватного режима

### DISCUSSION THREAD

**Discussion by NewLordZeeCaptain** [anticensority/runet-censorship-bypass#252] [Ideas]
Будет крайне удобно, если будет возможность выбрать, чтобы в приватном режиме был включен прокси на всех сайтах, а не только по списку. Примерно также как в FoxyProxy для Firefox

**Comment by ilyaigpetrov** [comment]
Прокси-сервера проекта https://antizapret.prostovpn.org:8443 работают только по реестру РКН.
Когда-то этого ограничения не было и некоторые пользователи или сервисы этим злоупотребляли: скрывали источник запросов за нашими серверами в целях взлома или хищений, обходили блокировку не в РФ, а в других странах, расширение FastProxy вообще просто использовало наши сервера против нашего несогласия.

Так что проксировать всё прокси-сервера проекта АнтиЗапрет не могут из-за защиты от злоупотребления.

Но если речь идёт о своих прокси или торе, то ваше замечание будет иметь возможность быть реализованным.
Можете разъяснить: чем проксирование всех адресов в приватном режиме удобно? Для чего?
